setup.py

```diff
@@ -2,55 +2,52 @@
 
 
 install_requires = [
-    'Django>=1.8,<2.3',
-    'tqdm==4.15.0',
-    'wagtail>=1.8,<2.9',
+    "Django>=1.8,<2.3",
```

Comment:
Should this be 1.11, since we're not testing against Django older than 1.11?

```suggestion
    "Django>=1.11,<2.3",
```

Reply:
Removed support for all versions before Django 1.11